src/main/java/org/aksw/simba/squirrel/components/WhiteListFrontierComponent.java

```diff
@@ -0,0 +1,69 @@
+package org.aksw.simba.squirrel.components;
+
+import java.io.File;
+import java.util.Map;
+
+import org.aksw.simba.squirrel.configurator.RDBConfiguration;
+import org.aksw.simba.squirrel.configurator.SeedConfiguration;
+import org.aksw.simba.squirrel.configurator.WhiteListConfiguration;
+import org.aksw.simba.squirrel.data.uri.filter.InMemoryKnownUriFilter;
+import org.aksw.simba.squirrel.data.uri.filter.KnownUriFilter;
+import org.aksw.simba.squirrel.data.uri.filter.RDBKnownUriFilter;
+import org.aksw.simba.squirrel.data.uri.filter.RegexBasedWhiteListFilter;
+import org.aksw.simba.squirrel.data.uri.serialize.java.GzipJavaUriSerializer;
+import org.aksw.simba.squirrel.frontier.impl.FrontierImpl;
+import org.aksw.simba.squirrel.queue.InMemoryQueue;
+import org.aksw.simba.squirrel.queue.RDBQueue;
+import org.aksw.simba.squirrel.rabbit.RPCServer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class WhiteListFrontierComponent extends FrontierComponent{
```

Comment:
We wanted to remove that one and add the (optional) usage of a white list to the "normal" ForntierComponent.